### DISCUSSION TITLE
Large Dataset Requirement For Analysis

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#1964] [Q&A] [answered]
Hello,

Hope you are doing fine. I need a dataset comprising millions of atoms for doing benchmarking, Currently, I have used the dataset from https://dplibrary.deepmd.net/ for Water and Copper. The reason I need I am not getting much difference between multi GPU and single GPU  time.

It would be great if I can get the data that is used in this paper - https://arxiv.org/abs/2005.00223

I don't know how to generate data doing AIMD simulation.  

Thanks

**Comment by wanghan-iapcm** [comment]
The lammps command replicate would help you generate large configurations.

**Reply by AnuragKr** [reply]
Thanks, @wanghan-iapcm  for the response. I think The lammps data generation would help create the initial configuration for MD simulation. But it will not be helpful in DeePMD training as those labeled data can be obtained from AIMD simulation by taking snapshots. Please let me know if it is possible via LAMMPS also but the problem will be How to get the corresponding label. And also do we need to consider equilibrium relaxed data or normal data?

**Comment by wanghan-iapcm** [comment, marked as answer]
Please note that the training data does not "comprising millions of atoms". We trained DP models from dataset of dozens of atoms, and did MD simulations of millions of atoms.

**Reply by AnuragKr** [reply]
Thanks for the clarification.
